src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const act of actions) {
```

Comment:
use destructure assignment 
```suggestion
  for (const { type, keysToRemove, ... } of actions) {
```